docs/source/api-reference.rst

```diff
@@ -47,25 +47,17 @@ Modules
 .. automodule:: cats.forecast
     :members:
 
-``cats.parsedata``
-^^^^^^^^^^^^^^^^^^
-
-.. automodule:: cats.parsedata
-    :members:
 
-``cats.timeseries_conversion``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``cats.optimise_starttime``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. automodule:: cats.timeseries_conversion
+.. automodule:: cats.optimise_starttime
     :members:
 
-
 Python objects
 --------------
 
 
```

Comment:
Do we need [this section](https://tlestang.github.io/cats/api-reference.html#python-objects)? It seems redundant with the module documentation above.

Reply:
Good point Thibault, we can get rid of it. Can you remove the section here in this PR, please Lincoln?